test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
you can delete this file

Reply:
I think you use the `generate` command to create the `user` model, that creates these files, then you change it to `admin` and now you are using `rspec`